service-loadbalancer/README.md

```diff
@@ -124,7 +124,6 @@ metadata:
   name: myservice
   annotations:
     serviceloadbalancer/lb.sslTerm: "true"
-    serviceloadbalancer/lb.aclMatch: "-i /t1 /t2"
```

Comment:
can i trouble you to give a better example (I know the readme didn't have a good example to begin with, so feel free to defer)? Perhaps just add a line saying: here's how you get the /foo, and here's how you keep it as /.